physionet-django/user/management/commands/utility.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from django.utils import timezone
+
+from user.models import CredentialApplication
+
+
+def get_pending__reference_response_credentialing_application(n, days, ignore_reminded_application=False):
+    """
+    Get n CredentialApplication that has been pending reference response for more than the given days.
+    """
+    today = timezone.now()
+    limit = today - timezone.timedelta(days=days)
+
+    # get all applications that have decision pending
+    applications = CredentialApplication.objects.filter(status=0)
```

Comment:
These queries/filters look like they could be combined. Do you prefer to keep them separate?

Reply:
Ah yes. i was trying to make it more readable.

But i think you are right, it doesnot make much of a difference, the queries are quite simple. let me refactor these